### ISSUE TITLE
Selenium message:unknown error: 'args' must be a list

### ISSUE THREAD

**Issue by maelle** [ropensci/RSelenium#160]
### Operating System
Windows
### Selenium Server version (selenium-server-standalone-3.0.1.jar etc.)
3.8.1
### Browser version (firefox 50.1.0, chrome 54.0.2840.100 (64-bit) etc.)
2.34
### Expected behaviour
I use code from [this SO answer](https://stackoverflow.com/questions/29861117/r-rvest-scraping-a-dynamic-ecommerce-page)

```r
link <- "https://www.pexels.com/search/bird/"

# open the webpage
remDr$navigate("https://www.pexels.com/search/bird/")

# scroll down
for(i in 1:20){      
  remDr$executeScript(paste("scroll(0,",i*10000,");"),
                      args = list())
  # be nice and wait
  Sys.sleep(3)    
}

```
I does not work as noted in this question:
https://stackoverflow.com/questions/46127818/executescript-from-rselenium-gives-error-with-args
### Actual behaviour

Error I get
```r
Selenium message:unknown error: 'args' must be a list
  (Session info: chrome=63.0.3239.84)
  (Driver info: chromedriver=2.34.522940 (1a76f96f66e3ca7b8e57d503b4dd3bccfba87af1),platform=Windows NT 10.0.16299 x86_64)

Error: 	 Summary: UnknownError
 	 Detail: An unknown server-side error occurred while processing the command.
	 Further Details: run errorDetails method
```

This works as noted in the SO thread.

```r
# scroll down
for(i in 1:20){      
  remDr$executeScript(paste("scroll(0,",i*10000,");"),
                      args = list())
  # be nice and wait
  Sys.sleep(3)    
}
```

**Comment by shinsukato** [comment]
I am seeing this error when trying to run through the Basics tutorial.

**Comment by laughing4090** [comment]
Hi . But here come a new issue if I use scroll down JS.  "No encoding supplied: defaulting to UTF-8.
Error in resContent[["status"]] : subscript out of bounds" This Issue has suffered me for more than one year. could you help to solve. Thanks

**Comment by juyeongkim** [comment]
Hi @maelle! I am not sure if you are still working on this, but  this issue is resolved in 2c31c4f

This was due to changes in httr package (removing empty list in body before sending a request): r-lib/httr@ce50e60

Selenium was expecting args as a list, but in the request, args has been removed from body, so it was throwing that error.

BTW, these are really cool :bird: photos!

**Comment by maelle** [comment]
Thank you for the update! 

The end product was less pretty https://masalmon.eu/2018/01/07/rainbowing/
